selfdrive/controls/lib/latcontrol_curvature.py

```diff
@@ -0,0 +1,48 @@
+import math
+
+from cereal import log
+from selfdrive.controls.lib.latcontrol import MIN_STEER_SPEED, STEER_ANGLE_SATURATION_THRESHOLD, LatControl
+from selfdrive.controls.lib.pid import PIDController
+
+
+class LatControlCurvature(LatControl):
```

Comment:
you can subclass `LatControlPID` and remove the init function here

Reply:
you can subclass again if you do something like https://github.com/commaai/openpilot/pull/26744/commits/d8ada72314ed2990527e3c13be1ae4cb1528b08f